servers/filesystem_server.go

```diff
@@ -59,7 +59,32 @@ func main() {
 }
 
 func GetRoot(w http.ResponseWriter, req *http.Request) {
-	w.Write([]byte(`Download and run the Linux command line client with
+	w.Write([]byte(`Welcome to cpass.
+
+You can use this Sandstorm grain for storing passwords or other secrets. The three best things
+about CrypTag:
+
+- You access it from a secure client on your own computer. (Right now, this is Linux-only.
+  I'm looking forward to changing that. Want to help?)
+
+- Data never goes out of sync. (All data is stored on the server.)
+
+- Searches are efficient. This is CrypTag's key idea: store secret information on a server,
+  with labels that the server can't understand, that can still be used for search!
+
+Overview of daily use
+---------------------
+
+$ cpass-sandstorm create twitter.com/myusername
```

Comment:
On first seeing this I wondered if there was some sort of Twitter integration going on, or if this was going to look up my PGP key via Keybase based on my Twitter handle, or what. But on reflection I think "twitter.com/myusername" is merely a string tag and any tag could be used here. I would either add more explanation, or use a more neutral example tag.